packages/amplify-codegen/src/commands/add.js

```diff
@@ -123,6 +123,8 @@ async function add(context, apiId = null, region = 'us-east-1') {
     } else {
       schema = getSDLSchemaLocation(apiDetails.name);
     }
+  } else if (apiDetails) {
+    schema = await downloadIntrospectionSchemaWithProgress(context, apiDetails.id, 'schema.json', region);
```

Comment:
Using introspection schema because the utility function already exists, but we could change to sdl schema.